package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "fiware-sth-rest-api",
+  "description": "Node.js app exposing the Short Time Historic (STH) FIWARE component REST API for reading",
+  "version": "0.0.1",
```

Comment:
0.0.1 or 0.1.0? If this values matches the next relase value, then I think it should be 0.1.0.